tests/e2e/scenarios/test_reborn_webui_v2_legacy_pending_messages.py

```
@@ -272,6 +272,61 @@ async def handle_delayed_send(route, _payload, fulfill_json):
         await harness["context"].close()
 
 
+async def test_reborn_legacy_send_failure_renders_inline_error_not_toast(
+    reborn_v2_server, reborn_v2_browser
+):
+    async def handle_failed_send(route, _payload, _fulfill_json):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `handle_failed_send`

Add return type annotation: `None`

(ANN202)